src/zimdump.cpp

```diff
@@ -291,6 +292,16 @@ void ZimDumper::writeHttpRedirect(const std::string& directory, const std::strin
     write_to_file(directory + SEPARATOR, outputPath, content.c_str(), content.size());
 }
 
+inline static bool testSymLink(const std::string& sym, const std::string& target) {
```

Comment:
`testSymlink` convey very few information. (What is tested ?)

It would be better to rename it to `checkSymlinkTarget` or `symlinkPointsTo`.